_conferences/droidcon-uk-2017.md

```diff
@@ -5,4 +5,7 @@ location: London, UK
 
 date_start: 2017-10-26
 date_end:   2017-10-27
+
+ cfp_end:   2017-05-14 
```

Comment:
Yep that should be easy. My other thought was to add a build step on Travis to verify since I've merged ones in the past without it.